src/cycax/cycad/assembly_blender.py

```diff
@@ -0,0 +1,126 @@
+
+    def _swap_xy_(self, rotation: tuple, rot: float, rotmax: tuple) -> tuple:
+        """Used to help rotate the object on the spot while freezing the top"""
+
+        while rot != 0:
+            max_y = rotmax[1]
+            rotation[0], rotation[1] = max_y - rotation[1], rotation[0]
+            rotmax[0], rotmax[1] = rotmax[1], rotmax[0]
+            rot = rot - 1
+        return rotation, rotmax
+
+    def _swap_xz_(self, rotation: tuple, rot: float, rotmax: tuple) -> tuple:
+        """Used to help rotate the object on the spot while freezing the front"""
+        while rot != 0:
+            max_x = rotmax[0]
+            rotation[0], rotation[2] = rotation[2], max_x - rotation[0]
+            rotmax[0], rotmax[2] = rotmax[2], rotmax[0]
+            rot = rot - 1
+        return rotation, rotmax
+
+    def _swap_yz_(self, rotation: tuple, rot: float, rotmax: tuple) -> tuple:
+        """Used to help rotate the object on the spot while freezing the left"""
+        while rot != 0:
+            max_z = rotmax[2]
+            rotation[1], rotation[2] = max_z - rotation[2], rotation[1]
+            rotmax[2], rotmax[1] = rotmax[1], rotmax[2]
+            rot = rot - 1
+        return rotation, rotmax
+
+    def _move(self, Rotmax: tuple, moves: tuple, Rotate: tuple):
+        """
+        Computes the moving and rotating of the stl to the desired location.
+
+        Args:
+            Rotmax: This is the tuple that contains the original (x,y,z) location.
+            moves: This is the tuple that contains the amount which the (x,y,z) needs to move by.
+            Rotate: This is the tuple that contains the amount which the (x,y,z) needs to be rotated.
+        """
+        rotation = [0, 0, 0]
+        for item in Rotate:
+            if item == 0:
+                bpy.ops.transform.rotate(value=math.radians(90), orient_axis="X")
+                working = self._swap_yz_(rotation, 1, Rotmax)
+
+            if item == 1:
+                bpy.ops.transform.rotate(value=math.radians(90), orient_axis="Y")
+                working = self._swap_xz_(rotation, 1, Rotmax)
+
+            if item == 2:
+                bpy.ops.transform.rotate(value=math.radians(90), orient_axis="Z")
+                working = self._swap_xy_(rotation, 1, Rotmax)
+
+            rotation = working[0]
+            Rotmax = working[1]
+
+        bpy.ops.transform.translate(value=(rotation[0] + moves[0], rotation[1] + moves[1], rotation[2] + moves[2]))
+
+    def _colour(self, colour: str, part: str) -> str:
+        """
+        Gives the colour.
+        Args:
+            colour: Colour which the object will become.
+        """
+        working_part = f"{part}_{self.parts[part]}"
+        template_object = bpy.data.objects.get(working_part)
+        matcolour = bpy.data.materials.new(colour)
+        matcolour.diffuse_color = (0, 1, 0, 0.8)
+
+        template_object.active_material = matcolour
+
+    def assembly_blender(self, path: Path | None = None):
```

Comment:
We know this is the blender class thus no need to add it to the method name. Call this `run`, `build`, `assemble` or something like that. Make sure it is the same entry point for all AssemblyEngines.